operations/firmware/Update_Firmware_with_FAS.md

```diff
@@ -57,29 +58,29 @@ The following table describes the hardware items that can have their firmware up
 
 <a name="order-of-operations"></a>
 
-### Order Of Operations
+## Order Of Operations
 
 For each item in the `Hardware Precedence Order`:
 
 1. Complete a dry-run:
 
-     1. `cray fas actions create {jsonfile}`
-     2. Note the ActionID!
-     3. Poll the status of the action until the action `state` is `completed`:
-        1. `cray fas actions describe {actionID} --format json`
+   1. `cray fas actions create {jsonfile}`
+   2. Note the ActionID!
+   3. Poll the status of the action until the action `state` is `completed`:
+      1. `cray fas actions describe {actionID} --format json`
 
-  2. Interpret the outcome of the dry-run. Look at the counts and determine if the dry-run identified any hardware to update:
+2. Interpret the outcome of the dry-run. Look at the counts and determine if the dry-run identified any hardware to update:
 
-     For the steps below, the following returned messages will help determine if a firmware update is needed. The following are end `state`s for `operations`. The Firmware `action` itself should be in `completed` once all operations have finished.
+   For the steps below, the following returned messages will help determine if a firmware update is needed. The following are end `state`s for `operations`. The Firmware `action` itself should be in `completed` once all operations have finished.
 
-     *	`NoOp`: Nothing to do, already at version.
-     *	`NoSol`: No viable image is available; this will not be updated.
-     *	`succeeded`:
-     	*	IF `dryrun`: The operation should succeed if performed as a `live update`. `succeeded` means that FAS identified that it COULD update a component name (xname) + target with the declared strategy.
-     	*	IF `live update`: the operation succeeded, and has updated the component name (xname) + target to the identified version.
-     *	`failed`:
-     	*	IF `dryrun`: There is something that FAS could do, but it likely would fail; most likely because the file is missing.
-     	*	IF `live update`: the operation failed, the identified version could not be put on the component name (xname) + target.
+   * `NoOp`: Nothing to do, already at version.
+   * `NoSol`: No viable image is available; this will not be updated.
+   * `succeeded`:
+      * IF `dryrun`: The operation should succeed if performed as a `live update`. `succeeded` means that FAS identified that it COULD update a component name (xname) + target with the declared strategy.
```

Comment:
```suggestion
      * IF `dryrun`: The operation should succeed if performed as a `live update`. `succeeded` means that FAS identified that it COULD update a component name (xname) and target with the declared strategy.
```